Cargo.toml

```diff
@@ -33,7 +33,7 @@ web-sys = { version = "0.3", features = [
 ] }
 
 openidconnect = { version = "3.0", optional = true }
-yew-nested-router = { version = ">=0.5, <0.7", optional = true }
+yew-nested-router = { version = ">=0.6.3, <0.7", optional = true }
```

Comment:
I guess it's time to bring this back to just `0.6.3`. But I can do this post-merge.